test/clients/test_clients.py

```diff
@@ -1,25 +1,35 @@
+from hypothesis import given
+from hypothesis.searchstrategy.lazy import LazyStrategy
+from hypothesis.strategies import lists, text
 import pytest
-
 from profiles.clients import Client, Clients
 
 
-def test_it_contains_clients():
-    client1 = Client('name1', 'client_id1', 'client_secret1', 'redirect_uri1')
-    client2 = Client('name2', 'client_id2', 'client_secret2', 'redirect_uri2')
-    clients = Clients(client1, client2)
+def list_of_str(length: int) -> LazyStrategy:
```

Comment:
Nice and simple to do.

Might be worth moving this elsewhere (`conftest.py`?). Maybe rename to `list_of_text` (as it's wrapping `text()`), or maybe even make it generic (`list_of(text(), 4)`) if that's possible?

Reply:
Upon further investigation, the preferred pattern seems to be just declaring the '@given' arguments explicitly rather than through helper functions. The '@fixture's play nice alongside '@given' but not so much with, which is fine as long as you are being explicit.

```
@given(lists(text(), max_size=4, min_size=4), lists(text(), max_size=4, min_size=4))
def test_it_contains_clients(client1_args, client2_args):
```

Reply:
Fair enough. :+1: